jsonschema-core.xml

```diff
@@ -2095,6 +2101,442 @@
             </section>
         </section>
 
+        <section title="Output Formatting" anchor="output">
+            <t>
+                JSON Schema is defined to be platform-independent.  As such, to increase compatibility
+                across platforms, implementations SHOULD conform to a standard validation output
+                format.  This section describes the minimum requirements that consumers will need to
+                properly interpret validation results.
+            </t>
+
+            <section title="Format">
+                <t>
+                    JSON Schema output is defined using JSON.  Implementations MAY deviate from this
```

Comment:
Can we say that it is defined with the JSON Schema instance data model as described in section 4.2.1?  That would make the convertable-to-JSON requirement very clear, as anything that fits that data model can be converted.

Reply:
✔️